common/src/main/java/tech/pegasys/web3signer/common/ApplicationInfo.java

```diff
@@ -10,7 +10,7 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package tech.pegasys.web3signer.commandline;
+package tech.pegasys.web3signer.common;
```

Comment:
Seems right to move this out of commandline. Wondering if it should be in core rather than common though?

Reply:
Core sounds like a better package to have the ApplicationInfo class although the PlatformDetector class that has been moved as well but it doesn't seem appropriate to me to move it to core.
Maybe move both under the "util" directory of the "core" package?

Reply:
Looks like PlatformDetector is only used by ApplicationInfo so probably makes sense to keep those together. 

I just noticed ApplicationInfo is shared between app and core, so common is fine too. The app module is a bit of an odd one, wonder if that should be rolled into core at some point?